drake/systems/plants/collision/bullet_model.cc

```diff
@@ -637,17 +639,15 @@ bool BulletModel::closestPointsAllToAll(
     const std::vector<ElementId>& ids_to_check, const bool use_margins,
     std::vector<PointPair>& closest_points) {
   std::vector<ElementIdPair> id_pairs;
-  for (auto idA_iter = ids_to_check.begin(); idA_iter != ids_to_check.end();
-       ++idA_iter) {
-    auto elementA_iter = elements.find(*idA_iter);
-    if (elementA_iter != elements.end()) {
-      for (auto idB_iter = idA_iter + 1; idB_iter != ids_to_check.end();
-           ++idB_iter) {
-        auto elementB_iter = elements.find(*idB_iter);
-        if (elementB_iter != elements.end()) {
-          if (elements[*idA_iter]->collidesWith(elements[*idB_iter].get())) {
-            id_pairs.push_back(std::make_pair(*idA_iter, *idB_iter));
-          }
+  for (int i = 0; i < ids_to_check.size(); ++i) {
```

Comment:
> In the past I was told not to use single-character variable names because that would prevent quick search-and-replace or multi-cursor highlighting in a text editor.

An interesting historical note -- thanks! I agree that modern IDEs don't have this problem.